packs/data/familiar-abilities.db/spell-delivery.json

```diff
@@ -3,34 +3,33 @@
     "img": "systems/pf2e/icons/actions/Passive.webp",
     "name": "Spell Delivery",
     "system": {
+        "actionType": {
+            "value": "passive"
+        },
+        "actions": {
+            "value": null
+        },
+        "category": "familiar",
         "description": {
             "value": "<p>If your familiar is in your space, you can cast a spell with a range of touch, transfer its power to your familiar, and command the familiar to deliver the spell. If you do, the familiar uses its 2 actions for the round to move to a target of your choice and touch that target. If it doesn't reach the target to touch it this turn, the spell has no effect.</p>"
         },
-        "duration": {
-            "expiry": "turn-start",
-            "sustained": false,
-            "unit": "unlimited",
-            "value": -1
-        },
         "level": {
             "value": 1
         },
```

Comment:
?

Reply:
I think I assumed it was unused actor data, I shall remove.